stdlib/smtplib.pyi

```diff
@@ -124,7 +124,7 @@ class SMTP:
         msg: _Message,
         from_addr: Optional[str] = ...,
         to_addrs: Optional[Union[str, Sequence[str]]] = ...,
-        mail_options: List[str] = ...,
+        mail_options: Sequence[str] = ...,
```

Comment:
Technically this could be Iterable (same for sendmail above), since the implementation just iterates over it once.